docs/randomblind.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Randomblind issuance
+---
+
+<script type="text/x-mathjax-config">
+  MathJax.Hub.Config({
+    extensions: ["tex2jax.js"],
+    jax: ["input/TeX", "output/HTML-CSS"],
+    tex2jax: {
+      inlineMath: [ ['$','$'], ["\\(","\\)"] ],
+      displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
+      processEscapes: true
+    },
+    "HTML-CSS": { fonts: ["TeX"] }
+  });
+</script>
+<script type="text/javascript" async src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js"></script>
+
+*Randomblind* issuance is a new IRMA feature introduced in
+[irmago v0.6.0](https://github.com/privacybydesign/irmago/releases/tag/v0.6.0)
```

Comment:
Use backticks here, i.e. `` `irmago` v0.6.0``, for consistency with other pages refering to irmago

Reply:
Maybe we should not include the version at all, since it will be listed on the v0.6.0 page of the documentation?

Reply:
I guess it still makes sense to add it, since it will show in 0.6.0 and above.